DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/register_map_and_streaming_interfaces/src/register_map_functor_model.cpp

```diff
@@ -0,0 +1,118 @@
+#include <sycl/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/ext/intel/prototype/interfaces.hpp>
+
+#include "exception_handler.hpp"
+
+using namespace sycl;
```

Comment:
Please remove this `using namespace sycl` and use `sycl::` explicitly in the code